cmd/run.go

```diff
@@ -78,6 +82,11 @@ func (r *KoolRun) Execute(originalArgs []string) (err error) {
 		return
 	}
 
+	if len(r.Flags.EnvVariables) > 0 && len(r.commands) > 1 {
+		err = ErrExtraEnvMultiLines
```

Comment:
we do error out when passing extra parameters to multi-line scripts... because parameters are most certainly signature aware so having them for multi-line script is assumed to be a mistake and possible source of problems...

but environment variables on the other hand do not affect the commands necessarily. So I guess for environment variables we are good to have them for all commands in a multi-line script.